src/oatpp/network/tcp/Connection.cpp

```diff
@@ -115,7 +115,7 @@ v_io_size Connection::write(const void *buff, v_buff_size count, async::Action&
 
   if(result < 0) {
     auto e = errno;
-    if(e == EAGAIN || e == EWOULDBLOCK){
+    if(e == EAGAIN){ // EAGAIN === EWOULDBLOCK
```

Comment:
Please revert this back to 
`if(e == EAGAIN || e == EWOULDBLOCK) ...`

We assume that somewhere may exist exotic platform where `EAGAIN` and `EWOULDBLOCK` may have different values

Reply:
I know of no platform that has these as different values.
I'll try to come up with a solution.
If that is not possible then fixing -Wlogical-op is not possible.

Reply:
I think the solution I came up is acceptable?

Reply:
Looks like we can do the following:

```cpp
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wlogical-op "

/* code */

#pragma GCC diagnostic pop
```

Reply:
will do that in its own PR.